Change project's bintray library

farbodsz opened the pull request:
Also corrects version numbers.
Should solve issues with Travis build

farbodsz committed:
Change project's bintray library

Also corrects version numbers.
Should solve issues with Travis build (updates #14)

FarbodSalamat-Zadeh commented:
Not yet - there are still issues as the library dependency 'fails to resolve' in Gradle.

farbodsz committed:
Minor formatting improvements

Add version badge
Improve the Groovy formatting
Remove copyright indent (in raw) and replace with triple `.

FarbodSalamat-Zadeh commented:
Note:
- Also added minor improvements to README

FarbodSalamat-Zadeh commented:
The issue has been fixed, so it should work.

FarbodSalamat-Zadeh commented:
And I forgot, this merge closes #14
